Hippo.Jobs.PuppetSync/appsettings.json

```diff
@@ -24,7 +24,7 @@
   "Puppet": {
     "RepositoryOwner": "ucdavis",
     "RepositoryName": "[external]",
-    "ReposotoryBranch": "main",
+    "RepositoryBranch": "main",
```

Comment:
Fixes typo, but is that going to break anything? How did it work before?

Reply:
The default of 'main' wasn't being overridden in any secrets or azure env variables.